src/libraries/System.Text.Json/tests/Serialization/StreamTests.cs

```diff
@@ -0,0 +1,355 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class StreamTests
+    {
+        // To hit all possible continuation positions inside the tested object,
+        // the outer-class spacer needs to be between 5 and 116 bytes long.
```

Comment:
Thanks -- doing padding like this makes sense and was previously discussed, but never implemented AFAIK.

Reply:
We added some to [test the parameterized ctors feature](https://github.com/dotnet/runtime/pull/33444/files#diff-2f4d68bf74486907056cc46853c29e89R150), but it would be good to extend these tests as suggested in https://github.com/dotnet/runtime/pull/42158#discussion_r488907272.